packages/kit/src/exports/vite/index.js

```diff
@@ -107,10 +107,14 @@ const warning_preprocessor = {
 		if (!filename) return;
 
 		const basename = path.basename(filename);
-		if (basename.startsWith('+layout.') && !content.includes('<slot')) {
+		if (
+			basename.startsWith('+layout.') &&
+			!content.includes(isSvelte5Plus() ? '{@render children(' : '<slot')
```

Comment:
This will catch `{@render children()`, but do we also need to catch things with different spacing?
```
{@render
    children()
```

Reply:
```js
let { children: meddlingKids } = $props();
```
```svelte
{@render meddlingKids()}
```